administrator/components/com_admin/script.php

```diff
@@ -310,6 +312,49 @@ private function migrateCompatPlugin($rowOld)
         )->execute();
     }
 
+    /**
+     * This method is for migration for old logrotation system plugin migration to task.
+     *
+     * @param   \stdClass  $data  Object with the extension's record in the `#__extensions` table
+     *
+     * @return  void
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+    private function migrateLogRotationPlugin($data)
+    {
+        /** @var SchedulerComponent $component */
+        $component = Factory::getApplication()->bootComponent('com_scheduler');
+
+        /** @var TaskModel $model */
+        $model = $component->getMVCFactory()->createModel('Task', 'Administrator', ['ignore_request' => true]);
+
+        if (!$data->enabled) {
+            return;
+        }
```

Comment:
```suggestion
        if (!$data->enabled) {
            return;
        }

        /** @var SchedulerComponent $component */
        $component = Factory::getApplication()->bootComponent('com_scheduler');

        /** @var TaskModel $model */
        $model = $component->getMVCFactory()->createModel('Task', 'Administrator', ['ignore_request' => true]);
```
No need to boot component and get model if nothing to do because disabled old plugin.